spec/fixtures/styled-components/bad.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Do these `import` lines need to be in here? They are causing ESLint to blow up CI.